src/openforms/formio/formatters/custom.py

```diff
@@ -21,10 +23,16 @@ def format(self, component: Component, value: str) -> str:
         return f"{fmt_date(parsed_value)} {fmt_time(parsed_value, 'H:i')}"
 
 
+type MapValue = PointGeometry | LineStringGeometry | PolygonGeometry
```

Comment:
this is the ideal compromise - it keeps the stuff that's not specific to map components in a generic place while still refining it sufficiently within the map component context